measurements-dash/dash.js

```diff
@@ -66,6 +66,14 @@ let bugLists = new Map([
       },
       columns: ["assigned_to", "summary"],
     }],
+    ["uplifts", {
+      category: "active",
+      searchParams: {
+        resolution: "---",
```

Comment:
Unfortunately this means we don't track closed bug here, which is the state we really need to track them somewhere.